packages/lit-html/src/lit-html.ts

```diff
@@ -819,7 +829,10 @@ export type Part =
   | ElementPart
   | EventPart;
 
-export class ChildPart {
+type Interface<T> = {[P in keyof T]: T[P]};
+
+export type ChildPart = Interface<ChildPartImpl>;
+class ChildPartImpl implements ChildPart {
```

Comment:
Is the `implements` necessary? Isn't this just "I implement myself"?

Reply:
oh oops, this is from when I had a hand-written interface